package.json

```diff
@@ -26,24 +29,33 @@
     "@babel/runtime": "^7.5.5",
     "axios": "^0.19.0",
     "babel-eslint": "^10.0.2",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.0.4",
+    "dotenv-webpack": "^1.7.0",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
     "eslint-loader": "^2.2.1",
+    "express": "^4.17.1",
+    "gh-pages": "^2.1.0",
```

Comment:
are you using gh-pages?

Reply:
@MCFrank16 reiterating Alain's feedback here

Reply:
no, I was just trying to deploy the project on github